JumpscaleCore/clients/sshkey/SSHKey.py

```diff
@@ -159,12 +159,13 @@ def is_loaded(self):
         :return: whether ssh key was loadeed in ssh agent or not
         :rtype: bool
         """
+        curr_pubkey = self.pubkey.split(" ")[1].strip()
```

Comment:
this is not needed, there is a property called `pubkey_only` in the ssh client object.

Reply:
This is sshkey client not ssh client so `pubkey_only` should not be available here right?

Reply:
yes `pubkey_only` should be used instead